Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -48,7 +48,7 @@ using GL = GraphicsContextGL;
 #if PLATFORM(COCOA)
 static GCEGLImage createAndBindCompositorTexture(GL& gl, GCGLenum target, GCGLOwnedTexture& texture, GL::EGLImageSource source)
 {
-    texture.ensure(gl);
+    texture.adopt(gl, gl.createTexture());
```

Comment:
At the moment this may fail if context has been lost, so you might do 
```
auto object = gl.createTexture();
if (!object)
   return nullptr;
texture.adopt(object);
```